android/app/src/main/java/org/cryptimeleon/incentive/app/ui/checkout/DsDetectedUi.kt

```diff
@@ -0,0 +1,71 @@
+package org.cryptimeleon.incentive.app.ui.checkout
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.cryptimeleon.incentive.app.ui.preview.CryptimeleonPreviewContainer
+
+@Composable
+fun DSPreventedUi(
+    navigateHome: () -> Unit = {},
+    disableDoubleSpending: () -> Unit = {},
+) {
+    Column(
+        modifier = Modifier
+            .padding(16.dp)
+            .fillMaxSize(),
+        verticalArrangement = Arrangement.SpaceBetween,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Column(
+            Modifier.weight(1f),
+            verticalArrangement = Arrangement.Center,
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            Text(
+                "🧐",
```

Comment:
this emoji seems quite underwhelming to me :)

Reply:
Do you have a recommendation for a more suitable one?